docs/sources/reference/components/mimir/mimir.rules.kubernetes.md

```diff
@@ -127,16 +129,38 @@ an `oauth2` block.
 [tls_config]: #tls_config-block
 [label_selector]: #label_selector-block
 [match_expression]: #match_expression-block
+[extra_query_matchers]: #extra_query_matchers-block
+[matcher]: #matcher-block
+
+### extra_query_matchers block
+
+The `extra_query_matchers` block has no attributes. It contains zero or more [matcher][] blocks.
+These blocks allow to add extra label matchers to all queries that are discovered by `mimir.rules.kubernetes`
+component. The algorithm of adding the label matchers to queries is the same as the one provided by 
+`promtool promql label-matchers set` command in [promtool](https://github.com/prometheus/prometheus/tree/main/cmd/promtool).
+
+### matcher block
+
+The `matcher` block describes a label matcher that will be added to each query found in `PrometheusRule` CRDs.
+
+The following arguments are supported:
+
+| Name         | Type     | Description                                        | Default | Required |
+|--------------|----------|----------------------------------------------------|---------|----------|
+| `name`       | `string` | Name of the label to match.                        |         | yes      |
```

Comment:
wouldn't it be more clear to call this `label`?

Reply:
I'm using names based on how they're called in the code. I think since "value" can also be called "label value" then "label name" and "name" are fine. A "label" in this sense consists of both "name" and "value". So if you're okay with it I'd like to leave it.